x-pack/legacy/plugins/maps/public/layers/sources/es_geo_grid_source/geo_tile_utils.js

```diff
@@ -37,15 +37,30 @@ function sinh(x) {
   return (Math.exp(x) - Math.exp(-x)) / 2;
 }
 
+// Calculate the minimum precision required to adequtely draw the box
+// bounds.
+//
+// ceil(abs(log10(tileSize))) tells us how many decimals of precision
+// are minimally required to represent the number after rounding.
+//
+// We add one extra decimal level of precision because, at high zoom
+// levels rounding exactly can cause the boxes to render as uneven sizes
+// (some will be slightly larger and some slightly smaller)
+function precisionRounding(v, minPrecision, binSize) {
```

Comment:
no need to pass minPrecision as a param
```suggestion
function precisionRounding(v, binSize) {
```

Reply:
I'm cool with this suggested change, but I'll describe why I made `minPrecision` an argument rather than using the global constant.  If the constant is used within the `precisionRounding` then the function cannot be easily reused or refactored for other purposes where a different minimum precision is desired.  I'll defer to whomever on the Kibana team has final say on which approach they prefer.